src/Element.php

```diff
@@ -195,28 +195,27 @@ public function setOption($key, $value)
      */
     public function setAttribute($key, $value)
     {
-        // Do not include the value in the list of attributes
-        if ($key === 'value') {
+        if ($key == 'value') {
+            // Do not include the value in the list of attributes
             $this->setValue($value);
-            return $this;
+        } else {
+            $this->attributes[$key] = $value;
         }
-        $this->attributes[$key] = $value;
         return $this;
     }
 
     /**
      * Retrieve a single element attribute
      *
-     * @param  $key
-     * @return mixed|null
+     * @param string $key
+     * @return mixed
      */
     public function getAttribute($key)
     {
-        if (! isset($this->attributes[$key])) {
-            return;
+        if ($key == 'value') {
```

Comment:
Please us a strict check here (`===`).

Reply:
I can’t imagine a situation in which this would make sense, can you showing example?

Reply:
@WinterSilence Check this out:
https://stackoverflow.com/questions/2401478/why-is-faster-than-in-php
also http://www.phpbench.com/

Reply:
@michalbundyra 1. it's true only when `$key` not string. 2. you convert type in any case - here or before pass in method. L - ogic (:

Reply:
Using anything other than a string for attribute names, either for storing the values or comparisons, is unlikely something we want to support. Usage of things like `SplString` or objects implementing `__toString()` for purposes such as this are pretty niche, particularly considering a standard use case is to use _configuration_ to seed your form elements.

So, while I understand your argument, in this context, it doesn't hold weight, and the decision to use strict comparisons is a reasonable one to make.